odl/solvers/util/callback.py

```diff
@@ -483,7 +492,17 @@ def __init__(self, *args, **kwargs):
         self.fig = kwargs.pop('fig', None)
         self.step = kwargs.pop('step', 1)
         self.saveto = kwargs.pop('saveto', None)
+        try:
+            self.saveto_formatter = self.saveto.format
+        except AttributeError:
+            self.saveto_formatter = self.saveto
```

Comment:
It seems it was caused by upgrading scipy to 0.19, at least on my local machine it works for scipy 0.18, but fails for 0.19.